packages/core-contracts/contracts/utils/AddressSet.sol

```diff
@@ -0,0 +1,70 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+library AddressSet {
+    struct Values {
```

Comment:
Can we have a better naming for the type?

i.e. what happens when we have a UintSet. The type can't also be called Values.

Reply:
Not adding the Set for now